From 307ee1e01bee8d90cac07bb2fb0abe498e3a3217 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cyril=20Bont=C3=A9?= <cyril.bonte@free.fr>
Date: Mon, 28 Sep 2015 23:16:06 +0200
Subject: [PATCH 01/10] DOC: documentation format cleanups

Chapter "Mailers" was missing from the summary, tabulations were used and
several lines exceeded 80 columns : some lines still exceeds the 80 columns
limit but are left as is, as it mostly concerns syntax and configuration
examples.
---
 doc/configuration.txt | 46 ++++++++++++++++++++++++----------------------
 1 file changed, 24 insertions(+), 22 deletions(-)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index f7f2bc5..a4cfd44 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -50,6 +50,7 @@ Summary
 3.3.      Debugging
 3.4.      Userlists
 3.5.      Peers
+3.6.      Mailers
 
 4.    Proxies
 4.1.      Proxy keywords matrix
@@ -654,9 +655,9 @@ deviceatlas-separator <char>
   is optional and set to | by default if not set.
 
 devicatlas-properties-cookie <name>
-  Sets the client cookie's name used for the detection if the DeviceAtlas Client-side
-  component was used during the request. This directive is optional and set to
-  DAPROPS by default if not set.
+  Sets the client cookie's name used for the detection if the DeviceAtlas
+  Client-side component was used during the request. This directive is optional
+  and set to DAPROPS by default if not set.
 
 external-check
   Allows the use of an external agent to perform health checks.
@@ -829,10 +830,10 @@ ssl-dh-param-file <file>
   ephemeral Diffie-Hellman (DHE) key exchange is used, for all "bind" lines
   which do not explicitely define theirs. It will be overridden by custom DH
   parameters found in a bind certificate file if any. If custom DH parameters
-  are not specified either by using ssl-dh-param-file or by setting them directly
-  in the certificate file, pre-generated DH parameters of the size specified
-  by tune.ssl.default-dh-param will be used. Custom parameters are known to be
-  more secure and therefore their use is recommended.
+  are not specified either by using ssl-dh-param-file or by setting them
+  directly in the certificate file, pre-generated DH parameters of the size
+  specified by tune.ssl.default-dh-param will be used. Custom parameters are
+  known to be more secure and therefore their use is recommended.
   Custom DH parameters may be generated by using the OpenSSL command
   "openssl dhparam <size>", where size should be at least 2048, as 1024-bit DH
   parameters should not be considered secure anymore.
@@ -2977,7 +2978,8 @@ email-alert from <emailaddr>
   and if so sending email alerts is enabled for the proxy.
 
   See also : "email-alert level", "email-alert mailers",
-             "email-alert myhostname", "email-alert to", section 3.6 about mailers.
+             "email-alert myhostname", "email-alert to", section 3.6 about
+             mailers.
 
 
 email-alert level <level>
@@ -7116,8 +7118,8 @@ source <addr>[:<port>] [interface <name>]
                 - 'ipv6@' -> address is always IPv6
                 - 'unix@' -> address is a path to a local unix socket
                 - 'abns@' -> address is in abstract namespace (Linux only)
-              You may want to reference some environment variables in the address
-              parameter, see section 2.3 about environment variables.
+              You may want to reference some environment variables in the
+              address parameter, see section 2.3 about environment variables.
 
     <port>    is an optional port. It is normally not needed but may be useful
               in some very specific contexts. The default value of zero means
@@ -11521,9 +11523,9 @@ da-csv-conv(<prop>[,<prop>*])
 
   Example:
     frontend www
-	bind *:8881
-	default_backend servers
-	http-request set-header X-DeviceAtlas-Data %[req.fhdr(User-Agent),da-csv(primaryHardwareType,osName,osVersion,browserName,browserVersion)]
+      bind *:8881
+      default_backend servers
+      http-request set-header X-DeviceAtlas-Data %[req.fhdr(User-Agent),da-csv(primaryHardwareType,osName,osVersion,browserName,browserVersion)]
 
 debug
   This converter is used as debug tool. It dumps on screen the content and the
@@ -13157,11 +13159,11 @@ rdp_cookie_cnt([name]) : integer (deprecated)
 req.ssl_ec_ext : boolean
   Returns a boolean identifying if client sent the Supported Elliptic Curves
   Extension as defined in RFC4492, section 5.1. within the SSL ClientHello
-  message. This can be used to present ECC compatible clients with EC certificate
-  and to use RSA for all others, on the same IP address. Note that this only
-  applies to raw contents found in the request buffer and not to contents
-  deciphered via an SSL data layer, so this will not work with "bind" lines
-  having the "ssl" option.
+  message. This can be used to present ECC compatible clients with EC
+  certificate and to use RSA for all others, on the same IP address. Note that
+  this only applies to raw contents found in the request buffer and not to
+  contents deciphered via an SSL data layer, so this will not work with "bind"
+  lines having the "ssl" option.
 
 req.ssl_hello_type : integer
 req_ssl_hello_type : integer (deprecated)
@@ -15418,10 +15420,10 @@ situations :
         stats socket ipv4@192.168.0.1:9999 level admin
         stats timeout 2m
 
-To access the socket, an external utility such as "socat" is required. Socat is a
-swiss-army knife to connect anything to anything. We use it to connect terminals
-to the socket, or a couple of stdin/stdout pipes to it for scripts. The two main
-syntaxes we'll use are the following :
+To access the socket, an external utility such as "socat" is required. Socat is
+a swiss-army knife to connect anything to anything. We use it to connect
+terminals to the socket, or a couple of stdin/stdout pipes to it for scripts.
+The two main syntaxes we'll use are the following :
 
     # socat /var/run/haproxy.sock stdio
     # socat /var/run/haproxy.sock readline
-- 
2.4.9

